v2/store/user.ts

```diff
@@ -179,6 +204,15 @@ const selectTokenDataById = (
 
   return state.user.accounts[account]?.data?.tokens[apiKey.toLowerCase()];
 };
+const selectBrineryDataById = (
+  state: RootState,
+  apiKey: string,
+  account: string | null | undefined,
+) => {
+  if (!account) return;
+
+  return state.user.accounts[account]?.data?.brineries.find((x) => x.assetKey === apiKey);
```

Comment:
good to lower/upper case these keys when doing lookups, we've had a bunch of bugs from that already

Reply:
I've also noticed that we're returning brineries as an array, which makes it clumsy to work with. It would be easier if they are an object indexed by api keys. That would also make it consistent with how we handle tokens:

<img width="456" alt="Screen Shot 2022-05-08 at 00 04 10" src="https://user-images.githubusercontent.com/7811733/167264502-8cf426ad-50ea-470f-9ec7-4b3b4e8a993b.png">